packages/contracts/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "contracts",
+  "packageManager": "yarn@4.1.0",
+  "scripts": {
+    "test": "forge test -vv",
+    "test:watch": "forge test -vv --watch",
+    "start": "anvil & copyfiles ../../.env contracts/.env && forge script script/*.sol --rpc-url http://localhost:8545 --broadcast -vv",
```

Comment:
the `.env` copying was required given that Foundry [can't handle env files on a custom path](https://github.com/foundry-rs/foundry/issues/5723)